nginx/development

```diff
@@ -16,7 +47,20 @@ server {
         proxy_redirect off;
     }
 
+    location ~ ^/(api|oidc) {
+        proxy_pass http://web;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Host $host;
+        proxy_redirect off;
+        access_log off;
+    }
+
     location / {
+        proxy_cache mycache;
+        add_header X-Cache-Status $upstream_cache_status;
```

Comment:
Would you be down to explain how this cache works in practice? How long it lasts, when it's cleared? I just don't understand nginx very well 😅 and I'm trying to make sure I get the difference between, say, displaying the same page to the viewer when it should have updated, versus sending a cached version to deal with bursts of activitypub traffic

Reply:
I'm not an nginx expert either... this is somewhat cargo culted based on various tutorials online.  The heuristic that I used was "cache it for a minute for any client that doesn't have a `session` cookie"

I've also found a problem with this in practice -- the cache key needs to be updated to include the `Accept:` header so that clients requesting `activitypub+json` get the correct format while HTML clients get the normal one.  I've patched this in my deployment and made the session detection better. I will try to make a better PR later today.

Reply:
Hopefully the updated PR makes these things clear? If not I'm happy to discuss further.

Reply:
Yes! Thank you so much -- I was missing the session cookie part of the equation.